src/components/ui/Checkbox.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "../../assets/css/Checkbox.css";
```

Comment:
maybe you need to create a mock data and loop on it to create a table row for each department